modules/datastore/modules/datastore_mysql_import/config/schema/datastore_mysql_import.schema.yml

```diff
@@ -0,0 +1,7 @@
+datastore_mysql_import.settings:
+  type: config_object
+  label: 'Datastore Mysql Import settings'
+  mapping:
+    remove_empty_rows:
+      type: boolean
+      label: 'Remove empty rows in dataset'
```

Comment:
More accurate to say 'Remove empty rows from the datastore table'

Reply:
Fixed